src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -328,6 +328,16 @@
             -n MyHttpSettings --probe MyNewProbe
 """
 
+helps['network application-gateway identity'] = """
+type: group
+short-summary: Manage the identity information
+"""
+
+helps['network application-gateway identity set'] = """
```

Comment:
To be consistent with rest, please use `assign`, you will also need to expose with `\remove\show``, please cross check `vm identity`